lib/msal-browser/docs/performance.md

```diff
@@ -50,3 +50,80 @@ const msalInstance = new msal.PublicClientApplication({
+
+**Note for MSFT first-party applications**: We will be publishing an internal build of `@azure/msal-browser` that is already instrumented to capture this telemetry. Contact us for more details.
+
+#### Example
+
+```typescript
+const msalInstance = new PublicClientApplication(config);
+
+msalInstance.addPerformanceCallback((events: PerformanceEvent[]) => {
+    events.forEach(event => {
+        console.log(event);
+    });
+});
+```
+
+Example event:
+
+```typescript
+const event: PerformanceEvent = {
+    correlationId: "03cad3ff-6682-4e3d-a0b4-d517b531c718",
+	durationMs: 1873,
+	endPageVisibility: "hidden",
+	fromCache: false,
+	name: "acquireTokenSilent",
+	startPageVisibility: "visible",
+	startTimeMs: 1636414041888,
+	success: true,
+    silentCacheClientAcquireTokenDurationMs: 0,
+    silentRefreshClientAcquireTokenDurationMs: 150,
+    silentIframeClientAcquireTokenDurationMs: 0
+    cryptoOptsGetPublicKeyThumbprintDurationMs: 200,
+    cryptoOptsSignJwtDurationMs: 8,
+    clientId: "b50703d7-d12b-4ddc-8758-91053fe0aba4",
+    authority: "https://login.microsoftonline.com/common",
+    libraryName: "@azure/msal-browser-1p",
+    libraryVersion: "2.22.2-beta.2"
+}
+```
+
+The complete details for `PerformanceEvents` objects can be found [here](../../msal-common/src/telemetry/performance/PerformanceEvent.ts). Below is a list of notable properties:
+
+| **Property**                       | Type      | Description                                                  |
+| ---------------------------------- | --------- | ------------------------------------------------------------ |
+| `name`                             | `string`  | Name of the operation, usually matches the top-level API name (e.g. `acquireTokenSilent`, `acquireTokenByCode`, `ssoSilent`). |
+| `durationMs`                       | `number`  | End-to-end duration in milliseconds for the operation.       |
+| `success`                          | `boolean` | Whether the operation was successful or not.                 |
+| `fromCache`                        | `boolean` | Whether the operation retrieved the result from the cache.   |
+| `correlationId`                    | `string`  | Correlation ID used for the operation.                       |
+| `libraryVersion`                   | `string`  | Version of MSAL.js used for the operation.                   |
+| `authority`                        | `string`  | Authority used for the operation.                            |
+| `<internalFunctionName>DurationMs` | `number`  | Duration in milliseconds for an internal operation.          |
+
+### removePerformanceCallback
+
+The `addPerformanceCallback` API will return a callback id, which an application can pass to `PublicClientApplication.removePerformanceCallback` to unregister that callback from receiving performance events. It will return a boolean indicating whether or the callback was successfully removed.
```

Comment:
It will return a boolean indicating whether or not the callback was successfully removed.